products/src/main/java/de/baleipzig/products/ProductController.java

```diff
@@ -0,0 +1,45 @@
+package de.baleipzig.products;
+
+import de.baleipzig.products.persistance.Product;
+import de.baleipzig.products.persistance.ProductDatabaseDummy;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+// Controller sitzt auf Port 80
```

Comment:
Nicht nur der Controller sitzt auf Port 80, sondern der gesamte Server.

Eingestellt ist das in der `application.properties`